spiffworkflow-frontend/src/routes/TaskShow.tsx

```diff
@@ -39,48 +39,55 @@ export default function TaskShow() {
     permissionRequestData
   );
 
+  const processResult = (result: any) => {
+    setTask(result);
+    const url = `/task-data/${modifyProcessIdentifierForPathParam(
+      result.process_model_identifier
+    )}/${params.process_instance_id}`;
+    if (
+      result.process_model_identifier &&
+      ability.can('GET', url)
+      // Assure we get a valid process model identifier back
+    ) {
+      HttpService.makeCallToBackend({
+        path: url,
+        successCallback: setUserTasks,
+        failureCallback: (error: any) => {
+          addError(error);
+        },
+      });
+    }
+  };
+
   useEffect(() => {
     if (permissionsLoaded) {
-      const processResult = (result: any) => {
```

Comment:
Why move this method out of the useEffect? useEffect generally doesn't like it when you move methods outside of it.